Slate RTL

mhrabiee opened the pull request:
Adding a simple _rtl.scss file which you can easily import to main screen.scss and have sufficient support of RTL styles packed with rtl custom font.
While it purposefully optimized for Persian (فارسی) language, it should be work fine with other rtl languages such as Arabic, Hebrew, etc.

lord commented:
This is a important addition and feature to have! Did you mean to close it? @mhrabiee

mhrabiee commented:
No no, I'm just wanted change some of the codes. Sorry i'm kinda beginner in open source contribution :grimacing:.

mhrabiee commented:
@lord Well first of all thank you from being supportive. I appreciate it. :pray:
- [x]  Don't include Persian typefaces — we'd want this change to be generic for all rtl fonts
**I just remove it. You are right it should be kept language agnostic** :+1:
- [x]  is there a reason we have the $right and $left variables instead of just using those things directly?
**I've included those variables for people who may wants change it to their preferred language, but i just realized that is absolutely redundant** :grin:

Again Thanks

lord commented:
Hey! Sorry for the slow reply, this is on my todo list, it's just taking some time! It may be another couple days, if that's alright :/

lord commented:
Still working, but so far my comments are:

- is there a reason we delete the fonts?
- any chance we could only have english in the readme? glad to include a comment about RTL text if it's written in English and under the 'features' section
- getting errors on things mentioning `vazir`

Starting to look pretty good, thanks so much again for working on this feature!

mhrabiee commented:
@lord Sorry for late reply. Thank you for your comments. :pray:

>  is there a reason we delete the fonts?

No, It's just noob mistake :grin: back it again.

> any chance we could only have english in the readme? glad to include a comment about RTL text if it's written in English and under the 'features' section

Reverted to English readme plus RTL Support in 'features'

> getting errors on things mentioning vazir

Yet another mistake, fixed.


Thank you so much.

mhrabiee commented:
Requested changes are committed.

lord commented:
Sorry for the incredibly slow reply — thanks for all your hard work! I've merged these changes in.

mhrabiee commented:
Thank you for being supportive and thanks for your incredible product. 👍
